src/main.scss

```diff
@@ -1,3 +1,17 @@
 html {
   scrollbar-color: rgba(var(--v-theme-on-background), var(--v-disabled-opacity)) rgb(var(--v-theme-background));
 }
+
+.splitpanes__pane {
+  background-color: transparent !important;
+  overflow-y: scroll !important;
+  padding: 0;
+
+  &:nth-of-type(3) {
+    overflow-y: hidden !important;
+  }
+}
+
+.splitpanes__splitter {
+  background-color: rgba(var(--v-theme-on-background), 0.05) !important;
```

Comment:
the `!important`s are awful. I don't have a better solution tho

Reply:
I know, I don't like them either. They only become a problem in cases though, were you later try to overwrite them in other situations. As we only use splitpanes in one situation I don't think it is that much of a problem right now to just do it like this.